Gemfile

```diff
@@ -3,17 +3,16 @@ source 'https://rubygems.org'
 gem 'rails', '4.1.5'
 
 gem 'awesome_print'
-gem 'activerecord-jdbc-adapter'
-gem 'activerecord-jdbcsqlite3-adapter'
-
+# gem 'activerecord-jdbc-adapter'
+gem 'activerecord-jdbcmysql-adapter'
+gem 'devise'
+gem 'doorkeeper'
 gem 'jdbc-mysql', :platform => :jruby
 gem 'jquery-rails'
 gem 'jruby-openssl'
 gem 'mysql2', :platform => :ruby
 gem 'plissken', '0.2.0'
-gem 'puma'
```

Comment:
No, we'll be using puma. I would use puma for development, running by foreman and Procfile.